make_profiler/report_export.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/python3
+
+from datetime import datetime
+DATE_FORMAT = "%Y-%m-%dT%H:%M:%SZ"
+
+
+def escape_json_string(s):
+    s=str(s)
+    s = s.replace("\"", "")
+    s = s.replace("'","")
+    s = s.replace("\\","\\\\") # single \ to double \\
+    return s
+
+
+def export_report(performance,docs):
+    strOutputFile = 'report.json'
+    fo = open(strOutputFile, 'w', encoding="utf-8")
+    n_in_progress=0
+    n_failed=0
+    n_total = 0
+    oldest_completed_target = ''
+    fo.write('{\n')
+    fo.write('    "status":[\n')
+    for key in performance:
+        rec=performance[key]
+        n_total += 1
+        if rec["failed"] == True:
```

Comment:
Checking for a boolean value is done with operator `is` if the value can have a type different from `bool`. Example:
```python
some_list = [True, False, 'text']
for x in some_list:
    if x is True:
        ...
```
But if the expected value can have only bool values, you can check it this way:
```python
some_list = [True, False, True, True]
for x in some_list:
    if x:
        ...
    elif not x:
        ...
```

Reply:
used if rec["failed"]: